src/main/java/com/passbook/sparkeighteen/controller/UserController.java

```diff
@@ -38,4 +40,10 @@ public ResponseEntity<LoginResponse> login(@Valid @RequestBody final LoginReques
         return new ResponseEntity<>(userService.login(request), HttpStatus.OK);
     }
 
+    @ApiOperation("delete user profile")
+    @DeleteMapping("/user/{userID}/profile")
```

Comment:
URL is wrong, you cannot delete a profile! You need to delete that user itself.

Reply:
updated url to @DeleteMapping("/user/{userID}")